docs/replication/group-replication-system-variables.md

```diff
@@ -27,8 +30,48 @@ The variable specifies a period of time in seconds before a node is automaticall
 
 In single-primary mode, the primary server ignores the timeout.
 
+## group_replication_certification_loop_chunk_size
+
+| Option | Description |
+|---|---|
+| Introduced | 8.0.32-24 |
+| Command-line |--group-replication-certification-loop-chunk-size |
+| Dynamic | Yes |
+| Scope | Global |
+| Data type | ulong |
+| Default value | 0 |
+
+Defines the size of the chunk that must be processed during the certifier garbage collection phase after which the client transactions are allowed to interleave. The default value is 0.
+
+The minimum value is 0. The maximum value is 4294967295.
+
+## group_replication_certification_loop_sleep_time
+
+| Option | Description |
+|---|---|
+| Introduced | 8.0.32-24 |
+| Command-line |--group-replication-certification-loop-sleep-time |
+| Dynamic | Yes |
+| Scope | Global |
+| Data type | ulong |
+| Default value | 0 |
+
+Defines the sleep time in microseconds that the certifier garbage collection loop allows client transactions to interleave. The default value is 0.
+
+The minimum value is 0. The maximum value is 1000000.
+
 ## group_replication_flow_control_mode
 
+| Option | Description |
+|---|---|
+| Introduced | 8.0.32-24 |
+| Command-line |--group_replication_flow_control_mode |
+| Dynamic | Yes |
+| Scope | Global |
+| Data type | Enumeration |
+| Default value | Quota |
+| Valid values | DISABLED <br> QUOTA <br> MAJORITY |
+
```

Comment:
Do we need this? We have the same information in line no 77.

Reply:
removed.